pkg/commands/environment.go

```diff
@@ -222,6 +222,7 @@ var environmentApplyCmd = &cobra.Command{
 
 		// if -namespace or a prNumber is provided, apply on given namespace
 		if optFlags.Namespace != "" || optFlags.PRNumber > 0 {
+			applier.Initialize()
```

Comment:
Not sure whey this is needed as applyNamespace instantiate a NewApply() here: https://github.com/ministryofjustice/cloud-platform-cli/blob/main/pkg/environment/environmentApply.go#L481